pkg/ddl/ddl_running_jobs.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Copyright 2013 The ql Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSES/QL-LICENSE file.
+
+package ddl
+
+import (
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/pingcap/tidb/pkg/parser/model"
+)
+
+type runningJobs struct {
+	sync.RWMutex
+	ids           map[int64][]model.InvolvingSchemaInfo
```

Comment:
Could we add another map(as revert index) using schema+tablen as the key and the "jobid1, jobid2" string as a value to make check conflict more efficient?

Reply:
OK.